{ Start Job wf_CLIMATE_FINANCE at Sun Nov 18 21:00:01 CET 2018
User Current Path: /home/undpccca/etl/wf_CLIMATE_FINANCE
Project name: CAMCLIMATE | Job name: Lib_ContextReader: context will be loaded from sub folder ./context/wf_CLIMATE_FINANCE.CAMCLIMATE_TEST.cfg
.----------------------+-----------------------------------------------------------------.
|                                   Context Parameters                                   |
|=---------------------+----------------------------------------------------------------=|
|key                   |value                                                            |
|=---------------------+----------------------------------------------------------------=|
|MySQL_Server          |localhost                                                        |
|MySQL_Database        |zadmin_zbiundpccca                                               |
|MySQL_Port            |3306                                                             |
|MySQL_Login           |zbiundpccca                                                      |
|MySQL_AdditionalParams|noDatetimeStringSync=true&useUnicode=true&characterEncoding=UTF-8|
|P_DATE_format         |dd-MMM-yyyy                                                      |
|P_Timestamp_format    |yyyy-MM-dd HH:mm:ss                                              |
|P_Default_String      |#                                                                |
|P_Default_Number      |0                                                                |
|P_Default_Date_String |01-Jan-1000                                                      |
|P_Default_Date_ID     |10000101                                                         |
'----------------------+-----------------------------------------------------------------'

===> Start Job m_cdc_s_odadata_frp at Sun Nov 18 21:00:02 CET 2018
Exception in component tDBConnection_1 (m_cdc_s_odadata_frp)
com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409)
	at com.mysql.jdbc.Util.getInstance(Util.java:384)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1013)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:987)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:973)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1112)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2506)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2539)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2321)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:832)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:409)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:417)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:344)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at routines.system.SharedDBConnection.getConnection(SharedDBConnection.java:63)
	at routines.system.SharedDBConnection.getDBConnection(SharedDBConnection.java:131)
	at camclimate.m_cdc_s_odadata_frp_0_1.m_cdc_s_odadata_frp.tDBConnection_1Process(m_cdc_s_odadata_frp.java:1626)
	at camclimate.m_cdc_s_odadata_frp_0_1.m_cdc_s_odadata_frp$2.run(m_cdc_s_odadata_frp.java:14736)
=== End Job m_cdc_s_odadata_frp at Sun Nov 18 21:00:02 CET 2018
Exception in component tRunJob_2 (wf_CLIMATE_FINANCE)
java.lang.RuntimeException: Child job running failed.
com.mysql.jdbc.exceptions.jdbc4.MySQLNonTransientConnectionException: Data source rejected establishment of connection,  message from server: "Too many connections"
	at camclimate.wf_climate_finance_0_1.wf_CLIMATE_FINANCE.tRunJob_2Process(wf_CLIMATE_FINANCE.java:2261)
	at camclimate.wf_climate_finance_0_1.wf_CLIMATE_FINANCE.tRunJob_1Process(wf_CLIMATE_FINANCE.java:1772)
	at camclimate.wf_climate_finance_0_1.wf_CLIMATE_FINANCE.tJava_1Process(wf_CLIMATE_FINANCE.java:959)
	at camclimate.wf_climate_finance_0_1.wf_CLIMATE_FINANCE$2.run(wf_CLIMATE_FINANCE.java:8296)
End Job wf_CLIMATE_FINANCE at Sun Nov 18 21:00:02 CET 2018  }
